src/notification.rs

```diff
@@ -106,21 +107,37 @@ mod change_lines_tests {
     }
 
     #[test]
-    fn same_host() {
+    fn multiple_on_single_host() {
```

Comment:
Nit: Multiple what? Elephants? 🐘
```suggestion
    /// If multiple changes are detected on a single host, then they are written one after the other. 
    fn multiple_change_notifications_on_single_host() {
```

Reply:
Tests are grouped in modules for this purpose.

https://github.com/EdJoPaTo/website-stalker/blob/b2ed3ecf9507006d7ec0691f98519712af9e3552/src/notification.rs#L89

(partial) output from `cargo test`:

```plaintext
test notification::change_lines_tests::different_hosts ... ok
test notification::change_lines_tests::mixed ... ok
test notification::change_lines_tests::multiple_on_single_host ... ok
test notification::change_lines_tests::multiple_per_host ... ok
test notification::change_lines_tests::single ... ok
test notification::commit_template_tests::empty_commit_always_empty ... ok
test notification::commit_template_tests::prefix ... ok
test notification::commit_template_tests::replace ... ok
test notification::e2e_with_commit ... ok
test notification::e2e_without_commit ... ok
```

Reply:
Oh yes! Of course. Still thinking in C language - where there is not go-to testing and everyone does something different :clown_face: